sslzen.com.cdn.json

```diff
@@ -0,0 +1,25 @@
+{
```

Comment:
There is file name vs. providerId/serviceId mismatch. Travis is failing. Pls correct either filename or serviceId.
`AssertionError: File name sslzen.com.cdn.json not matching the pattern. Expected sslzen.com.dns.json`

Reply:
@pawel-kow Fixed. Thank you.